libs/sdk-core/src/breez_services.rs

```diff
@@ -1591,11 +1603,19 @@ impl BreezServices {
             )
             .await?;
 
+        for swap in self.btc_receive_swapper.list_monitored()?.iter()
+            .filter(|swap| !swap.refundable())
+        {
+            let swap_address = &swap.bitcoin_address;
+            info!("Found non-refundable monitored swap with address {swap_address}, registering for swap tx notifications");
+            self.register_swap_tx_notification(swap_address).await?;
```

Comment:
If this call fails we may not try again next time since we already saved the new webhook url. I think we should move the `set_webhook_url` cal to the end of this function.